static/js/metrics-explorer.js

```diff
@@ -1432,137 +1537,113 @@ $('#json-block').on('click', function() {
 });
 
 // Merge Graphs in one
-function mergeGraphs(chartType, panelId=-1) {
-    if(isDashboardScreen){
```

Comment:
why did you remove this condition. This is being used in dashboards when we use metrics explorer.